reactor-netty-core/src/test/java/reactor/netty/ConnectionTest.java

```diff
@@ -513,11 +486,11 @@ private void doTestUnavailable(Mono<Void> publisher, String expectation) {
 		         })
 		         .block(Duration.ofSeconds(30));
 
-		assertNotNull(throwable.get());
+		assertThat(throwable.get()).isNotNull();
```

Comment:
you can actually chain assertions here without assigning to temp variable `t` first (including `throwable.get()` `isNotNull()` assertion above).